airbyte-integrations/connectors/destination-customer-io/src/test-integration/resources/expected-spec-oss.json

```diff
@@ -6,7 +6,34 @@
     "type" : "object",
     "additionalProperties" : true,
     "properties" : {
+      "credentials" : {
```

Comment:
Two things have changed in this spec:
* The ordering of properties (which shouldn't have any impact)
* The title of `object_storage_config` from `Object Storage Config` to `Object Storage Spec`

I think I'm fine with both changes above but wanted to make these explicit to the reviewer